api/test/test_edr.py

```diff
@@ -0,0 +1,186 @@
+import json
+from unittest.mock import patch
+
+import datastore_pb2 as dstore
+import routers.edr as edr
+from fastapi.testclient import TestClient
+from google.protobuf.json_format import Parse
+from main import app
+
+
+client = TestClient(app)
+
+
+def test_get_locations_id_with_single_parameter_query_without_format():
+    with patch("routers.edr.getObsRequest") as mock_getObsRequest:
+        test_data = load_json("test/test_data/test_single_proto.json")
+        compare_data = load_json("test/test_data/test_single_covjson.json")
+
+        mock_getObsRequest.return_value = create_mock_obs_response(test_data)
+
+        response = client.get(
+            "/collections/observations/locations/06260?"
+            + "parameter-name=ff&datetime=2022-12-31T00:00:00Z/2022-12-31T01:00:00Z"
+        )
+
+        # Check that getObsRequest gets called with correct arguments given in query
+        mock_getObsRequest.assert_called_once()
+        assert "ff" in mock_getObsRequest.call_args[0][0].filter["instrument"].values
+        assert "06260" in mock_getObsRequest.call_args[0][0].filter["platform"].values
```

Comment:
And only `620`

Reply:
https://github.com/EURODEO/e-soh/pull/30/commits/1ac4ad6138263bf82b8ca83b91571c18145611b4